Tasks/Stralenia/Task1/Task1.py

```diff
@@ -0,0 +1,16 @@
+import decimal
+
+a = decimal.Decimal(input("Начальная сумма:\n"))
+b = decimal.Decimal(input("Срок в годах:\n"))
+c = decimal.Decimal(input("Годовой процент:\n"))
+d = input("Ежемесячная капитализация (введите ДА или НЕТ):\n")
+
+x = a*(1+(c/(100*12)))**(12*b)
+
+y = a*(1+(c/100))**b
+
+if d == "ДА":
+    print("Итоговая сумма на счету: ", x)
+
+elif d == "НЕТ":
+    print("Итоговая сумма на счету: ", y)
```

Comment:
the russian language in the ui is not a good touch, but I will let it pass for now